checkUrlExists/template.yaml

```diff
@@ -25,15 +25,8 @@ Resources:
             Method: get
 
 Outputs:
-  # ServerlessRestApi is an implicit API created out of Events key under Serverless::Function
   # Find out more about other implicit resources you can reference within SAM
   # https://github.com/awslabs/serverless-application-model/blob/master/docs/internals/generated_resources.rst#api
-  LambdaApi:
```

Comment:
I don't understand exactly how serverless works in conjunction with CF, but won't you need to export a URL so that the dockstore stack can reference it and configure the web service with the location of the lambda?

Reply:
See comment above

Reply:
OK I've [exported the checkurlexists API url](https://github.com/dockstore/dockstore-deploy/blob/566184e455c74f9858588b68ea7b6503cfcde2e4/check-url-exists.yml.template#L126) from the CF template; is there an example of how to set the string in DockstoreWebserviceConfiguration.java via an import to dockstore.yml? Also need a hint on what to update in compose_setup

Reply:
> OK I've exported the checkurlexists API url from the CF template

Ah, I see.

> Also need a hint on what to update in compose_setup

Take a look at DATABASE_DOMAIN and ELASTICSEARCH_DOMAIN in compose_setup and dockstore-deploy.